core/Dockerfile

```diff
@@ -14,7 +14,7 @@ RUN apt-get -y update && apt-get install -y curl build-essential fastjar libmagi
 
 ### ADMIN (static build) ###
 WORKDIR /admin
-RUN curl -sL https://github.com/cheshire-cat-ai/admin-vue/releases/download/Admin/develop.zip | jar -xv
+RUN curl -sL https://github.com/cheshire-cat-ai/admin-vue/releases/download/Admin/release.zip | jar -xv
```

Comment:
This has to stay `develop.zip`

Reply:
Uhm... But i updated the fork before the PR... Uhm...

Reply:
Fixed with a revert of the merge commit, once again i don't have clue about why the update fork did that..